src/Containers/RightSideBar/Checklist.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Checkbox } from '../../Inputs/Checkbox';
+
+export interface DefCheckbox {
+    [key: string]: {
+        label: string;
+        value: boolean;
+    };
+}
+export interface CheckListProps {
+    title: string;
+    CRATitle: string;
+    ReqTitle: string;
+    RecTitle: string;
+    ReqInfoArr: DefCheckbox;
+    RecInfoArr: DefCheckbox;
+}
+
+export const RecInfoArr = {
```

Comment:
Make sure this info as well as ReqInfoArr are in storybook so other users can use this component as well